.phpcs.xml

```diff
@@ -9,7 +9,6 @@
   <!-- FIXME: original indentation is 4, but it was changed to 2 -->
   <exclude-pattern>./src/wovnio/modified_vendor</exclude-pattern>
   <!-- TODO: fix excluded files below -->
```

Comment:
this line should be removed as well?

Reply:
I'll remove later, in the indentation PR.

Reply:
ok

Reply:
thanks

Reply:
did by resolving conflicts actually ;)